src/wallet/coin_selection.rs

```diff
@@ -114,8 +114,9 @@ pub type DefaultCoinSelectionAlgorithm = BranchAndBoundCoinSelection;
 #[cfg(test)]
 pub type DefaultCoinSelectionAlgorithm = LargestFirstCoinSelection; // make the tests more predictable
 
-// Base weight of a Txin, not counting the weight needed for satisfaying it.
-// prev_txid (32 bytes) + prev_vout (4 bytes) + sequence (4 bytes) + script_len (1 bytes)
+/// Base weight of a Txin, not counting the weight needed for satisfying it
+///
+/// prev_txid (32 bytes) + prev_vout (4 bytes) + sequence (4 bytes) + script_len (1 bytes)
 pub const TXIN_BASE_WEIGHT: usize = (32 + 4 + 4 + 1) * 4;
```

Comment:
I think we could make this private (like with `pub(crate)`), since it's only used internally

Reply:
Makes sense `TXIN_BASE_WEIGHT` should be `pub(crate)` but that made it out of scope for the example code and I couldn't find a way to make it visible. As a work around I put a (hidden) duplicate const in the example. Hopefully this is OK or maybe someone knows a better way to fix it?